frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx

```diff
@@ -143,8 +142,8 @@ const getPayloads = (stepName: string, state: WizardState, hasOCS: boolean, t: T
       const { apiGroup, apiVersion, kind } = OCSServiceModel;
       const systemKind = getStorageSystemKind({ apiGroup, apiVersion, kind });
       return [
-        createStorageClusterPayload(state),
         createSSPayload(systemKind, StorageClusterSystemName),
```

Comment:
The reconciler handles creating Storage System right? Could we perhaps leverage that?

Reply:
@bipuladh We are indeed leveraging that. 
We are not creating the storage system for storage cluster. Updated